cmd/crc/cmd/status.go

```diff
@@ -217,6 +220,13 @@ func openshiftStatus(status *status) string {
 	return string(status.OpenShiftStatus)
 }
 
+func microShiftStatus(status *status) string {
+	if status.Preset == preset.Microshift {
```

Comment:
Shouldn't this be `if status.OpenShiftVersion != ""`? This makes the method identical to `openshiftStatus`?